samples/net/echo_client/boards/qemu_cortex_m3.conf

```diff
@@ -0,0 +1,10 @@
+# Select Stellaris Ethernet
```

Comment:
So after this it is not possible to use TAP device to connect to host system, is this really desired?

Reply:
Qemu's user mode networking can be used to connect to the host system. This is much simpler than SLIP+TAP. Also Qemu supports TAP, VDE, and socket backends as well.

Reply:
> Qemu's user mode networking can be used to connect to the host system.

You just disabled TAP with this config by default. To me it looks like you should place the Stellaris setup to an overlay config file. User can select that then if needed when compiling the application.